src/cf-buttons/usage.md

```diff
@@ -1,11 +1,20 @@
-- [Default button](#default)
-- [Secondary button](#secondary)
-- [Destructive action button](#destructive)
-- [Disabled button](#disabled)
-- [Super button](#super)
+- [Atoms](#atoms)
+  - [Default button](#default-button)
+  - [Secondary button](#secondary-button)
+  - [Destructive action button](#destructive-action-button)
+  - [Disabled button](#disabled-button)
+  - [Super button](#super-button)
+  - [Button links](#button-links)
+  - [Icon buttons](#icon-buttons)
```

Comment:
Maybe I could change the markup to look like how we do this molecule?
https://github.com/cfpb/cfgov-refresh/blob/flapjack/cfgov/unprocessed/css/molecules/form-field-with-button.less